console/src/schematic/slice.ts

```diff
@@ -401,23 +401,20 @@ export const { actions, reducer } = createSlice({
     fixThemeContrast: (state, { payload }: PayloadAction<FixThemeContrastPayload>) => {
       const { theme } = payload;
       const bgColor = new Color.Color(theme.colors.gray.l0);
+      const shouldChange = (crude: Color.Crude): boolean => {
+        const c = new Color.Color(crude);
+        return c.grayness > 0.85 && c.contrast(bgColor) < 1.3;
```

Comment:
Does this formula work well when going from dark to light instead of just light to dark?

Reply:
Updated grayness calc